app/reviews/models.py

```diff
@@ -1,5 +1,34 @@
 from __future__ import unicode_literals
 
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+from common.mixins import TimestampMixin
+from app.timetables.models import Serving
+
+
+class Review(TimestampMixin):
+    """
+    Model representing ratings and comments
+    submitted by users for menu item
+    they had on a particular day of the cycle.
```

Comment:
@andela-kerinoso let us follow the PEP way of writing multi-line docstrings. 
```
Multi-line docstrings consist of a summary line just like a one-line docstring,
followed by a blank line, 
followed by a more elaborate description.
```
https://www.python.org/dev/peps/pep-0257/#id17